jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/SubscribeSnsPanel.kt

```diff
@@ -32,7 +30,7 @@ class SubscribeSnsPanel(private val project: Project) {
     private fun createUIComponents() {
         topicSelector = ResourceSelector.builder()
             .resource(SnsResources.LIST_TOPICS)
-            .customRenderer(SimpleListCellRenderer.create { label, value, _ -> label.text = value.getName() })
+            .customRenderer(SimpleListCellRenderer.create("") { it.getName() })
```

Comment:
We have quite a lot of `SimpleListCellRenderer.create {` elsewhere in the code, is the issue just that it is null by default?

Reply:
yaaaa it could be issues else where, if we dont set a default and it returns > 1 result, the seelctedItem is null